modules/machineset-modifying.adoc

```diff
@@ -72,26 +122,49 @@ NAME                        PHASE     TYPE         REGION      ZONE         AGE
 +
 [source,terminal]
 ----
-$ oc annotate machine/<machine_name_original_1> \
+tag::CAPI[]
+$ oc annotate machine.cluster.x-k8s.io/<machine_name_original_1> \
+  -n openshift-cluster-api \
+  cluster.x-k8s.io/delete-machine="true"
+#CAPI values are guesses, need validation#
+end::CAPI[]
+tag::MAPI[]
+$ oc annotate machine.machine.openshift.io/<machine_name_original_1> \
   -n openshift-machine-api \
   machine.openshift.io/delete-machine="true"
+end::MAPI[]
 ----
 
 . To create replacement machines with the new configuration, scale the compute machine set to twice the number of replicas by running the following command:
 +
 [source,terminal]
 ----
 $ oc scale --replicas=4 \// <1>
-  machineset <machine_set_name> \
+tag::CAPI[]
+  machineset.cluster.x-k8s.io <machine_set_name> \
+  -n openshift-cluster-api
+end::CAPI[]
+tag::MAPI[]
+  machineset.machine.openshift.io <machine_set_name> \
   -n openshift-machine-api
+end::MAPI[]
 ----
 <1> The original example value of `2` is doubled to `4`.
 
 . List the machines that are managed by the updated compute machine set by running the following command:
 +
 [source,terminal]
 ----
-$ oc get -n openshift-machine-api machines -l machine.openshift.io/cluster-api-machineset=<machine_set_name>
+tag::CAPI[]
+$ oc get machines.cluster.x-k8s.io \
+  -n openshift-cluster-api \
+  -l cluster.x-k8s.io/cluster-api-machineset=<machine_set_name>
```

Comment:
The label is `cluster.x-k8s.io/set-name=<machine_set_name>`
```
$ oc get machines.c -n openshift-cluster-api --show-labels                                                                                                 
NAME            CLUSTER             NODENAME                                    PROVIDERID                              PHASE      AGE     VERSION   LABELS
capi-ms-z4bqv   zhsun-aws29-8ksnk   ip-10-0-31-228.us-east-2.compute.internal   aws:///us-east-2a/i-04e7b2cbd61fd2075   Running    5m26s             cluster.x-k8s.io/cluster-name=zhsun-aws29-8ksnk,cluster.x-k8s.io/set-name=capi-ms,machine.openshift.io/cluster-api-cluster=zhsun-aws29-8ksnk
```

Reply:
No that label for GCP